CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+### citus v11.0.0_beta2 (April 4, 2022) ###
+
+* Adds support for `DOMAIN` objects
```

Comment:
```suggestion
* Adds support for propagation of `DOMAIN` objects
```

Reply:
I guess we have a better alternative if we want to use the word `propagation` here.

How about

```suggestion
* Adds propagation of `DOMAIN` objects
```


This looks similar to the items below

```
* Adds propagation for foreign server commands

* Adds propagation of `TEXT SEARCH CONFIGURATION` objects

* Adds propagation of `TEXT SEARCH DICTIONARY` objects

...

* Adds propagation of `CREATE SCHEMA .. GRANT ON SCHEMA ..` commands

...

* Adds propagation of `SCHEMA` operations
```